lib/file-perms.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary
```

Comment:
OK, great then. The company code looks great to me.

I had one 2nd though about the java bindings. It's clear that we want the
plugins under the gpl, but the library part we might want to license that
under the lgpl to permit java plugins that would depend in gpl incompatible
libs.

In lib/file-perms.c
<https://github.com/balabit/syslog-ng/pull/822#discussion_r46446643>:

> @@ -1,6 +1,6 @@
>  /*
>   * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary

@bazsi <https://github.com/bazsi> I've submitted the whole thing as a
separate PR. I include the script used for transformation when a commit is
not done manually.
#823 <https://github.com/balabit/syslog-ng/pull/823>

—
Reply to this email directly or view it on GitHub
<https://github.com/balabit/syslog-ng/pull/822/files#r46446643>.